content/docs/1.0.0/tutorial/get-to-know-iofog.md

```diff
@@ -4,48 +4,44 @@ In this step of the tutorial we'll better familiarize ourselves with the ioFog e
-```
-
-For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard, try it out: http://localhost:10102/?load=dashboard.json
+For example, there's a microservice running an instance of [Freeboard](https://github.com/Freeboard/freeboard), an open source visualization dashboard for IoT devices. It's running a web server that serves the dashboard, try it out at http://localhost:10102/?load=dashboard.json
 
 The Freeboard microservice doesn't know it's running locally, so it could just as well be running on real edge node hardware!
 
 ## Shell Into Your Containers
 
-Our tutorial environment has four containers, the most important are: iofog-agent-1, iofog-agent-2, iofog-controller, and iofog-connector. As their names imply, they contain two [Agents](../agents/overview.html), a [Controller](../controllers/overview.html), and a [Connector](../connectors/overview.html).
+Our tutorial environment has three ioFog containers:
 
-You can think of each of these containers as if they were separate devices. In production, your Controller is most often running on a cloud server and your Agents are each running on individual edge hardware nodes in the field. The Controller is controlling the Agent the same way it would if the devices were hundreds of miles away, and the Connector can broker communication between any microservices you run.
+* iofog-agent (see [Agents](../agents/overview.html))
+* iofog-controller (see [Controller](../controllers/overview.html))
+* iofog-connector (see [Connector](../connectors/overview.html)
 
-To create a shell (bash) environment into one of your containers you can use the [`docker exec -ti`](https://docs.docker.com/engine/reference/commandline/exec/) command.
+You can think of each of these containers as if they were deployed on separate devices. In production, your Controller is most often running on a cloud server and your Agents are each running on individual edge hardware nodes in the field. The Controller is controlling the Agent the same way it would if the devices were hundreds of miles away, and the Connector can broker communication between any microservices you run.
 
-#### Agent's Containers
+To interact with your containers you can use the [`docker exec -ti`](https://docs.docker.com/engine/reference/commandline/exec/) command.
 
-Let's start a shell into one of the containers that is running one the two Agents.
+#### Agent's Container
 
-```sh
-docker exec -ti iofog-agent-1 bash
-```
-
-Now we're inside the first Agent container and can use the `iofog-agent` daemon to find out its status. Try this one:
+Let's start by using the `iofog-agent` CLI to find out its status. Note that the first `iofog-agent` in the following command stands for the container name, the second `iofog-agent` is the executable wrapped in the container.
```

Comment:
Note that the first `iofog-agent` in the following command stands for the container name**, and** the second `iofog-agent` is the executable wrapped in the container.